src/lib.rs

```diff
@@ -554,9 +542,9 @@ impl Adapter {
     /// # Panics
     ///
     /// Panics if the extensions specified by `desc` are not supported by this adapter.
-    pub fn request_device(&self, desc: Option<&DeviceDescriptor>) -> Device {
+    pub fn request_device(&self, desc: &DeviceDescriptor) -> Device {
```

Comment:
How are you supposed to call request_device(None) now?

Reply:
similarly, by doing `request_device(&wgpu::DeviceDescriptor::default())`